### DISCUSSION TITLE
Anyone won the last mission of the Price of Loyalty?

### DISCUSSION THREAD

**Discussion by HubKing** [ihhub/fheroes2#6877] [Q&A]
I mean the one with lots of islands on a large ocean. It seems impossible. The enemies are all seem allied and, even though the Wiki says that enemies don't go out of their islands, in Fheroes, they frequently come to my islands, even my starting castle to take it. And that's at week 1.

At the last try, I thought it was going well, and at the 3rd week of month 3, purple came with 61 black dragons and 21 black titans, and that was not even the only purple heroes (there was another purple hero with black dragons) and there were heroes of other colours, all with a sizeable army.

If the A.I. isn't stupid, I don't know how this can be winnable, since the islands I can have have very few resources (1 or two mines), I do not think it is physically possible to get 61 black titans by the 3rd week of 3rd month (that's 15 weeks; even if I get upgraded cloud castle from the beginning, which is impossible, I would only have 3 x 15 = 45) to fight 61 black dragons + 21 black titans.

**Comment by zenseii** [comment]
Hi, @HubKing. I've seen it be beat by The Shadow Rooster when even the design glitch of Red team getting Dimension Door happens. It was a very long haul but the short summary of it was to lure the AI to travel by boat and disembark on an island and then steal their boat.

On the latest fheroes2 version you can't just summon boat to steal it but there are other ways to accomplish this if you sacrifice a weak hero.

Here's the first upload:
https://www.youtube.com/live/ytVGA3B9_5s?feature=share


Now that Town Portal for the AI is a thing I can only imagine it potentially getting harder, but we've discussed making limitations on what spells the AI can get in certain campaign maps to fix such design flaws.

**Reply by HubKing** [reply]
I give it up. It is impossible to beat. I found that I can dimensional-jump from the bottom-right island entry to the bottom-left island. I entered there and take all the castles, and eventually eliminated all other players except red. Then I went to the bottom-right island and by then, red's main hero had over 1K bone dragons, 1K ghosts, 700 vampires, etc. I used dimensional jump to take all the 5 castles on that island, but the game did not end. I guess the end condition is eliminating red completely. But against 1K bone dragons + 1K ghosts, and all undead troops + resurrection magic, no strategy seemed to work. 

I trapped him in the top island where there is only a barbarian castle and a barbarian town, by taking his boat with a weak hero when he landed. I tried to skip a lot of turns eventually out number his troops by keep pressing e/enter, and somehow after a while he escaped to the bottom-left island. I have no idea how he could do it. He had dimensional jump but the island is isolated and had no place to jump out of it. Also, it seems that the red's main hero gains bone dragon and ghosts without any castle. Are they generated by some sort of artefact or something? By the time when he escaped, he had 2K bone dragons and 15K ghosts. Nothing worked, and my main hero was defeated with 1K bone dragons+ 700 black titans + 1K battle dwarfs + 8K skeletons, etc.

**Reply by Alucard648** [reply]
You are still very lucky. Most attempts end up being rushed at 4th week by main hero via DD. 1K bone dragons + 500 Black Titans + 750 Liches + legion of vampire lords + MASS SLOW!!! = instant death.
